src/generateSdk/astGenerator/TsAstGenerator.ts

```diff
@@ -142,6 +148,12 @@ export class AstGenerator implements AstGeneratorInterface {
                     }
                 }
                 if (!typeAtLocationPath) {
+                    if (escapedText == "Request") {
+                        return { type: AstNodeType.RequestType, name: escapedText };
+                    }
+                    if (escapedText == "Response") {
+                        return { type: AstNodeType.ResponseType, name: escapedText };
+                    }
```

Comment:
I don't think this is enough. The user could define an interface called Request which would satisfy this check, but result in a runtime error. We should use the type checker to determine if that type really is the interface defined in @types/node